download_and_create_reference_datasets/v03/write_combined_reference_ht.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+import argparse
+import os
+
+import hail as hl
+
+from hail_scripts.reference_data.combine import join_hts, update_existing_joined_hts
+from hail_scripts.reference_data.config import GCS_PREFIXES, AccessControl
+from hail_scripts.utils.hail_utils import write_ht
+
+COMBINED_REFERENCE_HT_PATH = (
+    'combined_reference/combined_reference.GRCh{genome_version}.ht'
+)
+DATASETS = [
+    'cadd',
+    'dbnsfp',
+    'eigen',
+    'exac',
+    'geno2mp',
+    'gnomad_genomes',
+    'gnomad_exomes',
+    'mpc',
+    'primate_ai',
+    'splice_ai',
+    'topmed',
+    'gnomad_genome_coverage',
+    'gnomad_exome_coverage',
+]
+VERSION = '1.0.0'
+
+
+def run(environment: str, genome_version: str, dataset: str):
+    checkpoint_path = os.path.join(
```

Comment:
Do we feel like it's better to have the checkpoint paths be named or unnamed?

An alternative here is to just have this be `gs://seqr-scratch-temp/{uuid.uuid4()}.ht`, which may be clearer about what we're actually doing here.

Reply:
I opted for this... it makes the `update_joined_ht` code cleaner to not have to dependency inject a path.